package.json

```diff
@@ -43,6 +43,7 @@
     "lodash.isequal": "^4.4.0",
     "lodash.throttle": "^4.1.1",
     "lodash.uniqby": "^4.4.0",
+    "prop-types": "^15.5.10",
```

Comment:
This does not help, since the warning we get is from libraries that are not updated.